### ISSUE TITLE
Element is not clickable at (49,580)

### ISSUE THREAD

**Issue by ofernander** [InstaPy/InstaPy#1185]
Selenium fails to find and click the "like" button after commit 89753ae. I changed the xpath to be more specific in finding the buttons and this seems to have solved the issue for now. Currently through my 2nd test run to see if it fails again. Anyone else getting the below error? 

`
+        "/html[@class='js logged-in client-root']/body/span[@id='react-root']/section[@class='_sq4bv _29u45']/main[@class='_8fi2q _2v79o']/div[@class='_4xng7']/div[@class='_hm7pe']/article[@class='_7hhq6 _622au _5lms4 _4kplh']/div[@class='_ebcx9']/section[@class='_hmd6j _8oo9w']/a[@class='_eszkz _l9yih']/span[@class='_8scx2 coreSpriteHeartOpen']")

+        "/html[@class='js logged-in client-root']/body/span[@id='react-root']/section[@class='_sq4bv _29u45']/main[@class='_8fi2q _2v79o']/div[@class='_4xng7']/div[@class='_hm7pe']/article[@class='_7hhq6 _622au _5lms4 _4kplh']/div[@class='_ebcx9']/section[@class='_hmd6j _8oo9w']/a[@class='_eszkz _l9yih']/span[@class='_8scx2 coreSpriteHeartFull']")`

## Expected Behavior
Like image, move onto next. 

## Current Behavior
Script crashes with the following error. 

selenium.common.exceptions.WebDriverException: Message: unknown error: Element <a class="_eszkz _l9yih" href="#" role="button" aria-disabled="false">...</a> is not clickable at point (49, 580). Other element would receive the click: <div class="_3dy6p">...</div>

## InstaPy configuration
Headless
Chromedriver 2.34.522913

**Comment by githubuser345** [comment]
More people and I have the same error. Check out this issue https://github.com/timgrossmann/InstaPy/issues/1167

**Comment by ofernander** [comment]
@maurits131 

I noticed that but I wasn't aware that it was the same issue since my issue specifically identifies the element that it fails to find. 

With the changes I made it seems to correct the issue of not being able to find the right element. I ran the script through two sessions liking over 400 post since I've made the changes and it didn't crash. 

Just wondering if the xpath changes I made are "correct" or not...
